modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/util/openToast.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/**
+ * This is an alternative lightweight implementation of openToast() method of
+ * frontend-js-web.
+ *
+ * Its main goal is to make Clay and React unnecessary when rendering the popup
+ * so that applications using SPA don't pull those dependencies.
+ * @param  {!String} type 'warn' or 'error'
+ * @param  {!String} title
+ * @param  {!String} message
+ * @return {!Object}
+ * An object containing a dispose field pointing to a function than can be
+ * invoked to close the popup.
+ */
+
+const CLASS_NAME_MAP = {
+	error: 'danger',
+	warn: 'warning',
+};
+const ICON_MAP = {
+	error: 'exclamation-full',
+	warn: 'warning-full',
+};
+
+export function openToast(type, title, message) {
+	const {spritemap} = Liferay.Icons;
```

Comment:
Can we use `import {getSpritemap} from '@liferay/frontend-icons-web';` instead?

Reply:
Of course!  :clap: 

Glad that we can use stuff via ESM instead of `Liferay` global object :-).